.eslintrc.json

```
@@ -23,5 +23,19 @@
     "eslint-plugin/test-case-shorthand-strings": "error",
     "node/no-unsupported-features": ["error", { "version": 6 }],
     "prettier/prettier": "error"
-  }
+  },
+  "overrides": [
+    {
+      "files": ["__tests__/*.js"],
+      "plugins": ["jest"],
+      "extends": [
+        "eslint:recommended",
+        "plugin:jest/recommended",
+        "plugin:prettier/recommended"
+      ],
```

We should keep this as `__tests__/.eslintrc.json` was

```suggestion
      "extends": ["plugin:jest/recommended"],
```

---

AFAIK, overrides wipes out the whole property, that's why the full set of extends is required